packages/picasso/src/AvatarUpload/story/index.jsx

```diff
@@ -0,0 +1,78 @@
+import PicassoBook from '~/.storybook/components/PicassoBook'
+import { AvatarUpload } from '../AvatarUpload'
+
+const page = PicassoBook.section('Components').createPage(
+  'AvatarUpload',
+  'Gets the image from file input and displays it in the Avatar component.'
+)
+
+page.createTabChapter('Props').addComponentDocs({
+  component: AvatarUpload,
+  name: 'AvatarUpload',
+  additionalDocs: {
+    onDropAccepted: {
+      name: 'onDropAccepted',
+      type: {
+        name: 'function',
+        description: '(files: File, event) => void',
+      },
+      description:
+        'Callback for when the drop event occurs. Note that if no files are accepted, this callback is not invoked.',
+    },
+    onDropRejected: {
+      name: 'onDropRejected',
+      type: {
+        name: 'function',
+        description: '(fileRejections: FileRejection, event) => void',
+      },
+      description:
+        'Callback for when the drop event occurs. Note that if no files are rejected, this callback is not invoked.',
+    },
+    validator: {
+      name: 'validator',
+      type: {
+        name: 'function',
+        description: '(file:File) => FileError | FileError[] | null',
+      },
+      description: 'Custom validation function',
+      defaultvalue: 'null',
+    },
+    onDrop: {
+      name: 'onDrop',
+      type: {
+        name: 'function',
+        description:
+          '(acceptedFiles: File, fileRejections: FileRejection, event:DragEvent | Event) => void',
```

Comment:
The accepted file can actually be null as well
```suggestion
          '(acceptedFile: File | null, fileRejection: FileRejection | null, event:DragEvent | Event) => void',
```